OONIProbeUITests/OONIProbeUITests.swift

```diff
@@ -54,7 +54,7 @@ class OONIProbeUITests: XCTestCase {
         app.navigationBars.buttons.element(boundBy: 0).tap()
 
         resultsButton.tap()
-        tablesQuery.cells.containing(.image, identifier:"performance").staticTexts["AS30722 - Vodafone Italia S.p.A."].tap()
+        tablesQuery.cells.containing(.staticText, identifier:"Performance").staticTexts["AS30722 - Vodafone Italia S.p.A."].tap()
```

Comment:
Why was this working before?

Reply:
before yes, on the new XCode the image element was not found.
Plus is more correct to look for a string not for an image.